MoTrPAC/motrpac-frontend
# Connects to #112. Connects to #113. Removing prior human meta-analysis implementation

### Key Changes

1. Removed prior implementation of human meta-analysis from **Human Analysis**
2. Fixed failing Storybook component of Analysis page
3. Fixed failing unit test due to Analysis page UI changes
4. Added external link out to the [ExtraMeta](https://extrameta.org) site on the **Useful Links** page

### Technical Notes

Testing this PR requires `node.js v14.15.4` and running `yarn install`

### Steps to test

1. Sign into your local test instance. Navigate to `^/analysis/human` and expect the previously active **Human Meta-analysis** card to be no longer present, while an inactive **Gene Time Course Clustering** card is now present.
2. Log out of the data hub portal and navigate to `^/external-links`. Expect to see the external link to `extrameta.org` to be present in the re-arranged UI of the top row.
3. Run `yarn storybook` and bring up the component instance on `port 9009`. Then navigate to the **Animal Depth 1** from the sidebar (e.g. http://192.168.68.112:9009/?path=/story/analysis-page--animal-depth-1). Expect the component to be rendered without errors.
4. Run `yarn test` and expect all tests to pass.
